_data/accessibility-tests/combo-box.yml

```diff
@@ -0,0 +1,97 @@
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Focus indicator is clearly visible around combo box.
+    summary_additional: When you tab into a combo box, a visible outline or other clear indication of focus appears around the combo box and options within.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: Tabbing into the combo box doesn't trigger a change of context.
+    summary_additional: When using a keyboard to access the combo box, there is no change until you activate with enter key or spacebar.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+  - summary: Keyboard actions can access all combo box functionality.
+    summary_additional: You can type, select, clear input and perform all other functions in a combo box element using only your keyboard.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+# Screen reader tests
+  - summary: Screen reader announces type and number of options.
+    summary_additional: When interacting with the combo box with a screen reader, you hear the component's label, the option names, and how many options there are.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.1
+  - summary: Screen reader announces "combo box" upon initial interaction. 
+    summary_additional: When you tab onto a combo box with a screen reader, you hear "combo box" along with instructions needed to complete the combo box.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.5
+  - summary: Screen readers identify combo boxes the same way when more than one is on a page.
+    summary_additional: When you tab to a combo box on a page that has other combo boxes, you will hear "combo box" every time you tab to a new combo box.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.4
+  - summary: Combo box offers visual and auditory instructions.
+    summary_additional: When you use the combo box component, you hear details about the purpose of the input with a screen reader and see them on the page. The instructions are specific to the form's intent (e.g., "Select a state", "Select a fruit").
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.2
+  - summary: Screen reader announces label, role, value, and controls for combo box. 
+    summary_additional: When using a screen reader with a combo box, you hear the label of the combo box, the role and the state. For example, label = Select an option, role = combo box, value = expanded. 
```

Comment:
**suggestion:** I feel like we need some kind of formatting on this. @finekatie I'll defer to your expertise, but here is one possibility of what I was thinking. Curious what you think. 

```suggestion
    summary_additional: When using a screen reader with a combo box, you hear the label of the combo box, the role and the state. For example, the label is announced as "Select an option", the role is announced as "combo box", and the value is announced as "expanded". 
```

Reply:
Preview:
![image](https://github.com/user-attachments/assets/a744020f-542a-4c2b-9c11-12ae01f0b01a)